packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -50,6 +50,25 @@ export type TrRoutingConfig = {
     // support it in both batch and single calculation
 };
 
+export type OsrmRoutingConfig = {
+    /**
+     * Set to null when using remote osrm server
```

Comment:
I think the comment are wrong here. 
We need the port if we use a remote OSRM

Reply:
I copied the comments from config_test.js. I'm not sure why the original says that.